docs/changes/visualization/14050.api.rst

```diff
@@ -0,0 +1,2 @@
+Passing a bare number as the ``coord_wrap`` argument to :meth:`astropy.visualization.wcsaxes.CoordinateHelper.set_coord_type` is deprecated.
+Pass a `~astropy.units.Qauntity` with units equivalent to angular degrees instead.
```

Comment:
Aren't we avoiding hyperlinks in the change log entirely?
```suggestion
Passing a bare number as the ``coord_wrap`` argument to ``wcsaxes.CoordinateHelper.set_coord_type`` is deprecated.
Pass a ``astropy.units.Quantity`` with angular units instead.
```

Reply:
Well... the debate is very old now but I think we did avoid hyperlink to avoid have to come back and change old change logs when we move things around (again) in the future.